tests/snapshots/assets__check_file@cetz-tree.typ-40.snap

```diff
@@ -142,9 +142,10 @@ input_file: tests/assets/cetz-tree.typ
     let children = ()
     let cnt = none
     if type(tree) == array {
-      children = tree.slice(1).enumerate().map((
-        (n, c),
-      ) => build-node(
+      children = tree
+        .slice(1)
+        .enumerate()
+        .map(((n, c)) => build-node(
         c,
         depth: depth + 1,
         sibling: n,
```

Comment:
not indented, why

Reply:
looks like we are having strang formatting for closures

Reply:
This is because when formatting `field access`, the last arg is not considered so it is not indented. Will resolve in next prs